examples/java/spring-boot/src/main/resources/log4j2-spring.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<Configuration status="WARN">
+    <Appenders>
+        <Console name="Console" target="SYSTEM_OUT" follow="true">
+            <JsonTemplateLayout eventTemplateUri="classpath:LogstashJsonEventLayoutV1.json">
+                <EventTemplateAdditionalField key="HOME_DIR" value="${env:HOME}"/>
```

Comment:
I think this was an based off an example from my blog post, right? If so, this isn't required, it was an example of how to add other metadata. I don't believe we need that produced into our logs

```suggestion
```